operators/src/source/gdal_source.rs

```diff
@@ -860,10 +881,11 @@ mod tests {
         };
 
         let time_interval_provider = TimeIntervalInformation {
-            time_intervals: vec![
-                TimeInterval::new_unchecked(1, 2),
-                TimeInterval::new_unchecked(2, 3),
-            ],
+            start_time: TimeInstance::from_millis(0),
+            time_step: TimeStep {
+                granularity: TimeGranularity::Seconds,
+                step: 1,
+            },
```

Comment:
Don't you have to start at 1 to get 1 as the first interval?
Moreover, it should be milliseconds.

Reply:
i changed the intervals in the tests so i think it is fine